.circleci/config.yml

```diff
@@ -86,54 +86,49 @@ jobs:
       - store_test_results:
           path: ./results
 
-  test_7:
+  test_zulu_7:
     <<: *default_test_job
     environment:
       - TEST_TASK: testJava7
 
-  test_8:
+  test_zulu_8:
     <<: *default_test_job
     environment:
-      # We are building on Java8, this is our default JVM so no need to set more homes
-      - TEST_TASK: test jacocoTestReport jacocoTestCoverageVerification
+      # default JDK provider is ZULU, hence home set
```

Comment:
I think this comment is very confusing here. What home? If Zulu is default provider, why `testJavaZULU8` and not just `testJava8` or `test`?

Reply:
Same as the other question - due to env vars naming confusion (zulu, openJdk, ...) I decided to put name of the provider with each job to make it clear which JDK is being used (while task names are autogenerated from env vars).